pkg/workceptor/kubernetes.go

```diff
@@ -703,6 +718,14 @@ func (kw *kubeUnit) SetFromParams(params map[string]string) error {
 	if userPod != "" && (userParams != "" || userCommand != "" || userImage != "") {
 		return fmt.Errorf("params kube_command, kube_image, kube_params not compatible with secret_kube_pod")
 	}
+
+	if pod_pending_timeout_string, ok := params["pod_pending_timeout"]; ok {
+		pod_pending_timeout, err := time.ParseDuration(pod_pending_timeout_string)
+		if err != nil {
+			return fmt.Errorf("Caould Not Parse pod_pending_timeout")
```

Comment:
let's put a logger.Error message, then return the err directly

```golang
logger.Error("Failed to parse pod_pending_timeout -- valid examples include '1.5h', '30m', '30m10s'")
return err
```

Reply:
I have made the required changes.